backup-parameter-store/src/main/scala/com/gu/aws/wrappers.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.aws
+
+import java.io.ByteArrayInputStream
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, InstanceProfileCredentialsProvider }
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.amazonaws.services.s3.model.{ ObjectMetadata, PutObjectRequest, PutObjectResult, SSEAwsKeyManagementParams }
+import com.amazonaws.services.simplesystemsmanagement.{ AWSSimpleSystemsManagement, AWSSimpleSystemsManagementClientBuilder }
+import com.amazonaws.services.simplesystemsmanagement.model.GetParametersByPathRequest
+
+import scala.collection.JavaConverters._
+import scala.annotation.tailrec
+
+object AwsConfig {
+  val provider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("frontend"),
+    InstanceProfileCredentialsProvider.getInstance())
+  val region = "eu-west-1"
+  val kmsKeyAlias: String = "arn:aws:kms:eu-west-1:642631414762:alias/FrontendConfigKey"
```

Comment:
Should this be an env var?

Reply:
It could be, although I don't think it will ever change